File: java/ProductionStatsService.java
import org.w3c.dom.Document;

public class ProductionStatsService implements StatsService {
    private ShoutyStatsService service = new ShoutyStatsService();

    public double getRevenue(int customerID) {
      /*
        String requestXml = "<Customer id=\"" + customerID + "\"/>";
        String responseXml = service.getRevenueForCustomer(requestXml);
        Document responseDocument = XmlHelper.parse(responseXml);

        return Double.parseDouble(responseDocument.getDocumentElement().getAttribute("revenue"));
      */

      throw new java.lang.UnsupportedOperationException("Uncomment code above and delete this exception");
    }
}

File: csharp-cd/ProductionStatsService.cs
using System.IO;
using System.Xml;

public class ProductionStatsService : IStatsService
{
    // private ShoutyStatsService service = new ShoutyStatsService();

    public decimal GetRevenue(int customerId)
    {
        /*
            string requestXml = "<Customer id=\"" + customerId + "\"/>";
            string responseXml = service.GetRevenueForCustomer(requestXml);

            var responseDocument = new XmlDocument();
            responseDocument.LoadXml(responseXml);
            return decimal.Parse(responseDocument.DocumentElement.Attributes["revenue"].Value);
        */

        throw new System.NotImplementedException("Uncomment code above and delete this exception");
    }
}
